x-pack/platform/plugins/shared/observability_ai_assistant/server/service/client/operators/get_generated_title.ts

```diff
@@ -55,7 +55,7 @@ export function getGeneratedTitle({
         {
           name: TITLE_CONVERSATION_FUNCTION_NAME,
           description:
-            'Use this function to title the conversation. Do not wrap the title in quotes',
+            'Use this function to title the conversation. Return only the title text without any labels, quotes, or prefixes like "Title:"',
```

Comment:
Do we never want the a prefix to be used?  Is it always undesirable or only when it's being literally with "Title" and by default?